docs/standard/exceptions/exception-class-and-properties.md

```diff
@@ -13,12 +13,10 @@ ms.author: "mairaw"
 
 The <xref:System.Exception> class is the base class from which exceptions inherit. For example, the <xref:System.InvalidCastException> class hierarchy is as follows:
 
-```
-Object
-  Exception
-    SystemException
-       InvalidCastException
-```
+Object  
```

Comment:
```suggestion
Object\
```

Reply:
@Youssef1313 Why are you suggesting a backslash at the end of these lines?

Reply:
For some reason, it won't render correctly in languages other than English, see #15689.

Reply:
We could also make these xref links like
```suggestion
<xref:System.Object>\
```

Reply:
@mairaw, You suggested to keep it in a code block without an id, but I think xrefs won't work inside code blocks

Reply:
See my comment below. I tried the xref suggestion.